pkg/sippyserver/server.go

```diff
@@ -66,9 +66,10 @@ type TestGridDashboardCoordinates struct {
 }
 
 type StandardReport struct {
-	CurrentPeriodReport sippyprocessingv1.TestReport
-	CurrentTwoDayReport sippyprocessingv1.TestReport
-	PreviousWeekReport  sippyprocessingv1.TestReport
+	CurrentPeriodReports []sippyprocessingv1.TestReport
+	CurrentPeriodReport  sippyprocessingv1.TestReport
+	CurrentTwoDayReport  sippyprocessingv1.TestReport
```

Comment:
If you precompute all reports, this become unnecessary, right?

Reply:
yep this was just a quick POC so I avoided too much refactoring, but we could definitely remove these